cloud_run_service_connect_cloud_sql/main.tf

```diff
@@ -0,0 +1,114 @@
+# [START cloudrun_service_cloudsql_dbuser_secret]
+resource "google_secret_manager_secret" "dbuser" {
+  project   = "your-project-id"
+  secret_id = "dbusersecret"
+  replication {
+    automatic = true
+  }
+}
+# [END cloudrun_service_cloudsql_dbuser_secret]
+
+# [START cloudrun_service_cloudsql_dbuser_secret_version]
+resource "google_secret_manager_secret_version" "dbuser1" {
+  secret      = google_secret_manager_secret.dbuser.id
+  secret_data = "secret-data"
+}
+# [END cloudrun_service_cloudsql_dbuser_secret_version]
+
+# [START cloudrun_service_cloudsql_dbpass_secret]
+resource "google_secret_manager_secret" "dbpass" {
+  project   = "your-project-id"
+  secret_id = "dbpasssecret"
+  replication {
+    automatic = true
+  }
+}
+# [END cloudrun_service_cloudsql_dbpass_secret_version]
+
+# [START cloudrun_service_cloudsql_dbpass_secret_version]
+resource "google_secret_manager_secret_version" "dbpass1" {
```

Comment:
It seems a bit confusing that the secret version resource name has 1 appended to it when it is actually adding it to google_secret_manager_secret.dbpass.id. WDYT about calling it `google_secret_manager_secret.dbpass` and `google_secret_manager_secret.dbpass_data`?